02-project-intro.Rmd

```diff
@@ -163,6 +163,13 @@ Now we have a good directory structure we will now place/save the data file in t
 >
 > We will load and inspect these data later.
 
+> ## Challenge 2 {.challenge}
+> It is usefull to get some general idea about the dataset, directly from the
+> command line, before loading it into R.
+> 1. Check on the command line how large is the file.
+> 2. How many rows of data does it contain?
+> 3. What kind of columns are there?
```

Comment:
Is this asking about the type of the columns (char, numeric, integer, etc)? Or what their names are? Or if they have names? Perhaps you could clarify this in the question since students might be confused.